### PR TITLE
Tweak packages structure

### PR DESCRIPTION
**Pull request by mandrigin**
1. Move `profiling` out of `helpers` because it was a single subpackage.
2. Set correct package name to `testing.go`.

### CODE REVIEW

File: profiling/heap.go [hunk 1]
```diff
@@ -0,0 +1,30 @@
+package profiling
```
Comment (themue) [inline profiling/heap.go:1]: Nice separation. 👍

**Review by themue — APPROVED** [review]
LGTM

**Review by adambabik — APPROVED** [review]

**Comment by divan** [comment]
Has anyone used this package? AFAIR, it was created and never used. I think it can be removed completely.

**Comment by mandrigin** [comment]
@divan profiling is referenced in 2 places, exported as C functions so they are usable from outside. I have no idea if they are used though.

**Comment by divan** [comment]
Yeah, it was created here https://github.com/status-im/status-go/pull/264, but I'm not aware if someone has used it. At least, when I needed profiles, I was just using benchmarks or quickly instrumenting code myself with pprof Start/Stop functions.

**Pull request merged**